CHANGELOG.md

```diff
@@ -1,6 +1,35 @@
 # Unreleased
 
-Nothing Yet!
+This release is mostly several fixes to how we create GitHub Releases, as well as some internal improvements for future feature work.
+
+
+## GitHub Release Ordering
+
+We now prefer creating your [GitHub Release in the host step](https://opensource.axo.dev/cargo-dist/book/reference/config.html#github-release), ensuring published npm and Homebrew packages never refer to URLs that don't yet exist.
```

Comment:
It may be worth us going into the details of how it used to work and why we made the change, since we punted on including that level of detail in the docs.